docker-compose example/docker-compose.example.yml

```diff
@@ -104,11 +116,26 @@ services:
     container_name: photoview-mariadb
     restart: unless-stopped
     stop_grace_period: 5s
+    user: "999:999"
```

Comment:
Why would we need this user IDs definition for the DB service on a PROD setup? The DB is not exposed by default to the world, it creates and manages the DB-related files under root ownership which is additional protection from some unintended modification on the host system.
Do you see a real need/benefit in this change?

Reply:
@kkovaletp setting this user id makes the container's entrypoint not modify the database folders to an arbitrary UID (like 0 for root)  here: https://github.com/MariaDB/mariadb-docker/blob/11135d071fd1fe355b1f7fa99b9d3b4a59bb5225/docker-entrypoint.sh#L200-L221
Doing this negates the requirement to add the `CHOWN` capability.

I think your point is valid for the scenario where ab user is mapping a host folder to the DB folder (instead of using a volume)

Reply:
Mapping a host folder as a volume to Photoview services is the default option (part of the examples) because in this way users have more control over hosting these volumes on their host file systems depending on the available limitations and use case.

Reply:
@kkovaletp I've replaced the hardcoded user with the necessary capabilities in https://github.com/photoview/photoview/pull/992/commits/82d317f00b817e945117dd4ea2f7bd4b9e2c6248